c_emulator/riscv_sim_RV32.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+		<key>CFBundleDevelopmentRegion</key>
+		<string>English</string>
+		<key>CFBundleIdentifier</key>
+		<string>com.apple.xcode.dsym.riscv_sim_RV32</string>
+		<key>CFBundleInfoDictionaryVersion</key>
+		<string>6.0</string>
+		<key>CFBundlePackageType</key>
+		<string>dSYM</string>
+		<key>CFBundleSignature</key>
+		<string>????</string>
+		<key>CFBundleShortVersionString</key>
+		<string>1.0</string>
+		<key>CFBundleVersion</key>
+		<string>1</string>
+	</dict>
```

Comment:
Delete

Reply:
This content is not edited by me, it is generated automatically

Reply:
You need to add *.plist to your. gitignore to ensure that these don't get into the repo

Reply:
It might be a better idea to add it to [$GIT_DIR/info/exclude](http://git-scm.com/docs/gitignore) instead. gitignore is supposed to be common between all contributers, I am not sure how practical is it to add all various editors and IDEs to gitignore.

Reply:
`*.plist` is a bad idea, those can be source files. You want the `*.dSYM` as a whole. Which really belongs in your machine's global gitignore file.